packages/tenant-process/src/services/tenantService.ts

```diff
@@ -424,6 +426,95 @@ export function tenantServiceBuilder(
       return updatedTenant;
     },
 
+    async revokeCertifiedAttributeById(
+      {
+        tenantId,
+        attributeId,
+        organizationId,
+        correlationId,
+      }: {
+        tenantId: TenantId;
+        attributeId: AttributeId;
+        organizationId: TenantId;
+        correlationId: string;
+      },
+      logger: Logger
+    ): Promise<void> {
+      logger.info(
+        `Revoke certified attribute ${attributeId} to tenantId ${tenantId}`
+      );
+      const requesterTenant = await retrieveTenant(
+        organizationId,
```

Comment:
```suggestion
      }: {
        tenantId: TenantId;
        attributeId: AttributeId;
      },
      { authData, correlationId, logger }: WithLogger<AppContext>
    ): Promise<void> {
      logger.info(
        `Revoke certified attribute ${attributeId} to tenantId ${tenantId}`
      );
      const requesterTenant = await retrieveTenant(
        authData.organizationId,
```
This is how you can get everything from the context

Reply:
I don't like to pass the entire `authdata` here, I prefer to pass only the `organizationId`, as for the other fields it's fine